src/js/step.js

```diff
@@ -266,16 +270,10 @@ export class Step extends Evented {
   hide() {
     this.trigger('before-hide');
 
-    if (this.el) {
-      this.el.hidden = true;
-      // We need to manually set styles for < IE11 support
-      this.el.style.display = 'none';
-    }
-
     document.body.removeAttribute('data-shepherd-step');
 
     if (this.target) {
-      this.target.classList.remove('shepherd-enabled', 'shepherd-target');
+      this._removeClassesFromTarget();
```

Comment:
Can we please just leave this as `classList.remove`? Moving one line into a function is actually adding more code, not to mention it's harder to read because now I have to search for this function to see what it is doing.
```suggestion
      this.target.classList.remove('shepherd-enabled', 'shepherd-target');
```

Reply:
True... but the reason I moved this to a function is because I'm calling it from two separate locations now.

Reply:
Yeah, but it doesn't save any lines of code, it adds lines, since it was just one line. I prefer to keep it as it was.

Reply:
Updated